lib/SlackBot.js

```diff
@@ -2,6 +2,7 @@ var Botkit = require(__dirname + '/CoreBot.js');
 var request = require('request');
 var express = require('express');
 var bodyParser = require('body-parser');
+var querystring = require('querystring');
```

Comment:
@sundeepgupta I believe you were using the built in querystring instead of NPM module. You don't need the package.json modification for that.